config/samples/ironic_v1beta1_ironic.yaml

```diff
@@ -22,4 +22,7 @@ spec:
     replicas: 1
     containerImage: quay.io/podified-antelope-centos9/openstack-ironic-inspector:current-podified
     pxeContainerImage: quay.io/podified-antelope-centos9/openstack-ironic-pxe:current-podified
-  secret: osp-secret
+  secret: ironic-secret
```

Comment:
is ironic-secret only for standalone? So this should remain osp-secret?

Reply:
ooops, that looks like a rebase conflict resolving mistake on my part.
I will update.